linux_os/guide/services/mail/postfix_harden_os/postfix_server_cfg/postfix_server_relay/postfix_prevent_unrestricted_relay/oval/shared.xml

```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="postfix_prevent_unrestricted_relay" version="1">
+    <metadata>
+      <title>Configure Postfix Against Unrestricted Mail Relay</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>Red Hat Enterprise Linux 8</platform>
+		<platform>multi_platform_wrlinux</platform>
+      </affected>
+      <description>Protect against unrestricted relay of mail.</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion comment="prevent unrestricted mail relay" test_ref="test_prevent_unrestricted_relay" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+  comment="Set banner" id="test_prevent_unrestricted_relay" version="1">
+    <ind:object object_ref="obj_postfix_restrict_relay" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="obj_postfix_restrict_relay" version="1">
+    <ind:filepath>/etc/postfix/main.cf</ind:filepath>
+    <ind:pattern operation="pattern match">^[\s]*smtpd_client_restrictions[\s]*=[\s]*permit_mynetworks,+reject[\s]*$</ind:pattern>
```

Comment:
Current regex would allow something like:

``smtpd_client_restrictions = permit_mynetworks,,,,reject``

Was that intended?

Reply:
(because of the ``,+reject``)

Reply:
Not intended. I did not realize the regex would allow that.